go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -164,9 +164,9 @@ func (ovn *Controller) getLogicalSwitchesForLoadBalancer(lb string) ([]string, e
 // TODO: Add unittest for function.
 func (ovn *Controller) generateACLName(lb string, sourceIP string, sourcePort int32) string {
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
-	aclName = strings.ReplaceAll(aclName, ":", "\\:")
+	aclNameTmp := strings.ReplaceAll(aclName, ":", "\\:")
 	// ACL names are limited to 63 characters
-	if len(aclName) > 63 {
+	if len(aclNameTmp) > 63 {
```

Comment:
is it actually the escaped version that has to be <= 63 characters or the unescaped version?

Reply:
@Billy99 added this so I'll let him respond.

Reply:
the esacpes don't matter for the total length being <= 63 so will fix this